src/main/java/org/folio/auth/authtokenmodule/tokens/TokenValidationException.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.auth.authtokenmodule.tokens;
+
+/**
+ * When a token is provided and it hasn't passed validation, for any reason,
+ * a TokenValidationException will be present. Token validation exceptions have
+ * an opinion about what http response code should be returned to clients.
+ */
+public class TokenValidationException extends Exception {
+  /**
+   * Returns the HTTP response code to return when this exception is encountered.
+   * @return The HTTP response code.
+   */
+  public int getHttpResponseCode() {
+    return httpResponseCode;
+  }
+  private int httpResponseCode;
```

Comment:
I can find variables more easily if they are all before the first method. https://dev.folio.org/guidelines/contributing/#coding-style , https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html#1852

Reply:
I'm fine with this if it is what the standard is on the project. Personally I like getters and privates to be grouped.